content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+deteriorating relative to policy.
+
+Policies don't necessarily have to be related to application security and
+vulnerabilities. You can use policies to measure and track other aspects of
+supply chain management as well, such as base image dependencies and
+open-source licenses.
+
+## Default policies
+
+Docker Scout ships the following three out of the box policies:
+
+- [Critical and high vulnerabilities with fixes](#critical-and-high-vulnerabilities-with-fixes)
+- [Critical vulnerabilities](#critical-vulnerabilities)
+- [Packages with GPL3+ licenses](#packages-with-gpl3-licenses)
+
+These policies are turned on for your active repositories by default. There's
+currently no way to turn off or configure these policies.
+
+### Critical and high vulnerabilities with fixes
+
+This policy requires that your artifacts aren't exposed to known
+vulnerabilities with a critical or high severity, and where there's a fix
+version available. Essentially, this means that there's an easy fix that you
+can deploy for images that fail this policy: upgrade the vulnerable package to
+a version containing a fix for the vulnerability.
+
+This policy is unfulfilled if an artifact is affected by one or more
+vulnerability where a fix version is available.
+
+### Critical vulnerabilities
+
+This policy requires that your artifacts contain no known critical
+vulnerabilities. The policy is unfulfilled if your artifact contains one or
+more critical vulnerabilities.
+
+This policy flags all critical vulnerabilities, whether or not there's a fix
+version available.
+
+### Packages with GPL3+ licenses
+
+This policy requires that your artifacts don't contain packages distributed
+under a GPL3+ [copyleft](https://en.wikipedia.org/wiki/Copyleft) license.
+
+This policy is unfulfilled if your artifacts contain one or more packages with
+a violating license.
+
+## View policy status
+
+The **Overview** tab of the [Docker Scout Dashboard](https://scout.docker.com/)
+displays a summary of recent changes in policy for your repositories.
+This summary shows images that have seen the most change in their policy
+evaluation between the most recently analyzed tag and the previous tag.
+
+![Policy overview](../images/policy-overview.png)
+
+### Policy status per repository
+
+The **Images** tab shows the current policy status, and recent policy trend,
+for all images in the selected environment. The **Policy status** column in the
+list shows:
+
+- Number of fulfilled policies versus the total number of policies
+- Recent policy trends
+
+![Policy status in the image list](../images/policy-image-list.png)
+
+The policy trend, denoted by the directional arrows, indicates whether a tag is
+better, worse, or unchanged in terms of policy, compared to the previous tag in
+the same environment.
+
+- The green arrow pointing upwards shows the number of policies that got better
+  in the latest version of this image.
```

Comment:
**thought:** Unsure if `latest version` is the right term here. Mainly because of my limited understanding. We've got
- latest pushed
- latest indexed

How does latest version correlate to the above two? I believe it's closer to latest pushed but in fact we are showing the latest indexed tags on the image page. 

CC. @chrispatrick WDYT?

Reply:
Latest version isn't necessarily wrong, but maybe a bit vague. Latest pushed image might be better.